<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEntrance</class>
 <widget class="QDialog" name="DialogEntrance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Вход</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/images/images/Launchpad 2.png</normaloff>:/new/images/images/Launchpad 2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSignIn">
      <attribute name="title">
       <string>Авторизация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="6" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>15</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="buttonSignIn">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #507299;
border-radius: 4px;
color:#FFFFFF;</string>
           </property>
           <property name="text">
            <string>Войти</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="boxLogin_in">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLineEdit" name="lineLogin_in">
              <property name="styleSheet">
               <string notr="true">border: #FFFFFF;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="boxPassword_in">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLineEdit" name="linePassword_in">
                <property name="styleSheet">
                 <string notr="true">border: #FFFFFF;</string>
                </property>
                <property name="inputMask">
                 <string/>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Normal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelHide_in">
                <property name="styleSheet">
                 <string notr="true">color: #507299;</string>
                </property>
                <property name="text">
                 <string>| Скрыть</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QGroupBox" name="boxLabelError_in">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: #FFFFFF;</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_10">
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="labelError">
                <property name="styleSheet">
                 <string notr="true">color:#BF0A00;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSignUp">
      <attribute name="title">
       <string>Регистрация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QGroupBox" name="boxPassword_up">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLineEdit" name="linePassword_up">
                <property name="styleSheet">
                 <string notr="true">border: #FFFFFF;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelHide_up">
                <property name="styleSheet">
                 <string notr="true">color: #507299;</string>
                </property>
                <property name="text">
                 <string>| Скрыть</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="buttonSignUp">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #507299;
border-radius: 4px;
color:#FFFFFF;</string>
           </property>
           <property name="text">
            <string>Зарегистрироваться</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="boxLogin_up">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QLineEdit" name="lineLogin_up">
              <property name="styleSheet">
               <string notr="true">border: #FFFFFF;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="boxLabelError_up">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: #FFFFFF;</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="labelError_up">
                <property name="styleSheet">
                 <string notr="true">color:#BF0A00;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineLogin_in</tabstop>
  <tabstop>linePassword_in</tabstop>
  <tabstop>buttonSignIn</tabstop>
  <tabstop>linePassword_up</tabstop>
  <tabstop>lineLogin_up</tabstop>
  <tabstop>buttonSignUp</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
